### PR TITLE
All jest test passing with mock authentication

### PR DESCRIPTION
**Pull request by charisearter**
## Description ##

Clearly documents the reason for the pull request and what has changed. Delete this test and add your own.
- [ ] This is a bugfix.
- [ ] This is a new feature.
- [x] This is part of a feature.

## Organization ##

- [x] Have you linked this pull request to a Trello card?

## Review ##

- [x] Have you added two reviewers to this pull request?
- [ ] Is this a work in process?
- [x] Is this ready to be reviewed?

### CODE REVIEW

File: __tests__/routes/watchlist.test.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+const Profiles = require('../../api/profile/profileModel');
+const profileRouter = require('../../api/profile/profileRouter');
+const watchlistRouter = require('../../api/watchlist/watchlistRouter');
+
+const server = express();
+server.use(express.json());
+
+jest.mock('../../api/profile/profileModel');
+jest.mock('../../api/watchlist/watchlist-bridgesModel');
+jest.mock('../../api/bridge/bridgeModel');
```
Comment (VinceWilliamsDev) [inline __tests__/routes/watchlist.test.js]: I don't think this endpoint is used anymore.  Line 12 can probably be removed.

**Review by VinceWilliamsDev — CHANGES_REQUESTED** [review]
Excellent work, please check my comment about line 12.

File: __tests__/routes/watchlist.test.js (hunk 1 above)
Comment (charisearter) [reply __tests__/routes/watchlist.test.js]: Ah. Okay. I'll do that now.

Comment (charisearter) [reply __tests__/routes/watchlist.test.js]: @VinceWilliamsDev Done!

**Review by Joe-Bay — APPROVED** [review]
It looks pretty good to me, good job

**Review by VinceWilliamsDev — APPROVED** [review]

**Pull request merged**